docker/docker.go

```diff
@@ -28,7 +33,7 @@ func AirflowCommand(id, airflowCommand string) string {
 	}
 
 	stringOut := string(out)
-	return stringOut
+	return stringOut, nil
```

Comment:
Noticed we're changing the function signature here and it got me wondering why this function didn't already return an error. Should we also return the error here on L32? - `return errors.Wrapf(err, "error encountered executing airflow command")`  or something?